docs/apache-airflow-providers-mysql/index.rst

```diff
@@ -39,13 +39,13 @@ Content
     :hidden:
     :caption: System tests
 
-    System Tests <_api/tests/system/providers/mysql/index>
+    System Tests <_api/tests/providers/mysql/index>
```

Comment:
not sure if this one is correct 🤔 would appreciate confirmation

Reply:
This one mentions system tests specifically so the `system` one is (probably) correct.

Reply:
Yes @fritz-astronomer  - I think what we need to fix is the "old" link generated directly in the generated [arifow-site](https://github.com/apache/airflow-site/tree/main/docs-archive/apache-airflow-providers-mysql) instead. 

This one should stay as it is.